docs/using_eessi/building_on_eessi.md

```diff
@@ -67,7 +67,32 @@ Finally, you should be able to load our newly build module:
 module load netCDF/4.9.0-gompi-2022a
 ```
 
-## Manually building software op top of EESSI
-Building software on top of EESSI would require your linker to use the same system-dependencies as the software in EESSI does. In other words: it requires you to link against libraries from the compatibility layer, instead of from your host OS.
+## Manually building software op top of EESSI (without EasyBuild)
+
```

Comment:
@ocaisa Which way do you think it's the best to add this piece of information? Should we replace the runtime part line where 
```
export LD_LIBRARY_PATH=$LIBRARY_PATH:$EBROOTGCCcore/lib64

```
is mentioned or should we add it in a separate section as an untested feature?

Reply:
It's already tested in the sense that we've tried it and it works. I imagine there are probably corner cases where it will not work. I would be inclined to replace the use of `LD_LIBRARY_PATH` as this is a much more straightforward solution.

Reply:
just pushed a commit with this changes: [ed0c5dd](https://github.com/EESSI/docs/pull/175/commits/ed0c5dd839e380f1ba4188d7ccb6c2ea48f37b0f)